config-templates/aws/aws-ebs-csi-driver/0.8.2/README.md

```diff
@@ -1,131 +1,98 @@
-## How to use the template to deploy aws-ebs-csi-driver on a Satellite Cluster?
+# AWS EBS CSI Driver
 
-### Features supported
+AWS EBS CSI driver implements the CSI specification for container orchestrators to manage the lifecycle of Amazon EBS volumes.
+
+**Features supported:**
 - Dyanamic Provisioning
 - Volume Resizing
 - Volume Snapshot 
 
-### Pre-reqs
-#### Planning consideration for Infra Admin
--  For better IOPS and throughput, use instance built on *Nirto System*. Refer to 
+## Prerequisites
+**Planning consideration for Infra Admin**
+-  For better IOPS and throughput, use instance built on *Nitro System*. Refer to 
 https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ebs-volume-types.html
 
-#### Planning consideration for Location Admin
+**Planning consideration for Location Admin**
 - The driver requires IAM permission to talk to Amazon EBS to manage the volume on user's behalf. Create an IAM user with proper permission and get *AWS Access Keys*. Refer to https://docs.aws.amazon.com/powershell/latest/userguide/pstools-appendix-sign-up.html
 
 - Volume type *io2* requires iopsPerGB, default is set to *10 IOPS/GB*.
   To set different value for iopsPerGB, refer to https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ebs-volume-types.html
```

Comment:
Let's put this sentence in the parameter table with the `iops` parameter - and include a statement that says they should pass the IOPs value that is required for the volume that type that they want to use.

Reply:
Moved to desciption column. Also `iops-per-gb` is required only by `sat-aws-block-gold` storageclass (volume type: io2). So I added note for same.
`sat-aws-block-gold storageclass (Volume type io2) requires iopsPerGB, default is set to 10 IOPS/GB. To set different value for iopsPerGB, refer to https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ebs-volume-types.html
`